docs/docs/getting-started/installation/minikube.md

```diff
@@ -51,7 +51,7 @@ The base URL for the [API](../../api/introduction.md) is the same to access the
 To customize your Minikube instance, you can adjust the `driver`, `cpus` and `memory` attributes in the following manner:
 
 ```
-airy create --provider=minikube --provider-config driver=virtualbox,cpus=4,memory=8192 my-airy
+minikube -p airy-core start --driver=docker --cpus=3 --memory=7168 --container-runtime=containerd --ports=80:80 --extra-config=apiserver.service-node-port-range=1-65535
```

Comment:
We don't need to change this, as the `airy` CLI also provides a way to customize the installation with the `--provider-config` flags.